src/warc2zim/content_rewriting/css.py

```diff
@@ -23,14 +23,30 @@ def rewrite(self, content: str | bytes) -> str:
             rules = parse_stylesheet(content)
         self.process_list(rules)
 
-        output = serialize(rules)
+        try:
+            output = serialize(rules)
+        except Exception:
+            import sys  # Import sys only in case of error
+
+            # If we cannot parse the css, simply return the original content
+            print("Error rewriting CSS", file=sys.stderr)  # noqa
```

Comment:
Please use a logger instead of print. 

It looks like it has not yet been used in this file, so you will have to `from warc2zim.constants import logger` first.

I suggest these logs are written only at "DEBUG" log level.